### ISSUE TITLE
Training classification accuracy >1

### ISSUE THREAD

**Issue by bahetibhakti** [CBICA/GaNDLF#351]
**Describe the bug**
Hi, I am training a classification model on MRI brain scans using vgg16 in gandlf. We expect to have accuracy in the range of 0-1. But Training classification accuracy is displayed >1. Validation accuracy seems correct. Attaching the screenshot below. Can you please have a look?

**Screenshots**
![Screenshot from 2022-03-17 14-13-02](https://user-images.githubusercontent.com/6376680/158874773-97c9a7cd-49a1-45fa-85c9-86b93533911b.png)

**GaNDLF Version**
0.0.14

**Comment by sarthakpati** [comment]
It seems like this line: https://github.com/CBICA/GaNDLF/blob/64d024a5892a4e06975a959cdc935f3d579bbd31/GANDLF/compute/training_loop.py#L179

Should be taking care of the any `batch_size` related inconsistencies. Any idea where this could be coming from, @orhunguley or @Geeks-Sid?

**Comment by Geeks-Sid** [comment]
@bahetibhakti Can you add a config file as well?

**Comment by sarthakpati** [comment]
> @bahetibhakti Can you add a config file as well?

I have emailed one of the configs to you and @orhunguley. @bahetibhakti this is the one with the `exp` scheduler. Is that okay?

**Comment by bahetibhakti** [comment]
Yes.  I had just edited the "q" parameters in that

**Comment by sarthakpati** [comment]
> Yes. I had just edited the "q" parameters in that

Oh okay, that shouldn't make a difference for the accuracy anyway.

**Comment by orhunguley** [comment]
hi @bahetibhakti , can you check if that [branch](https://github.com/orhunguley/GaNDLF/tree/master) solves your issue? I tried with some MRI data and in my case it seem like fixed.
